beacon-chain/core/state/transition_no_verify_sig.go

```diff
@@ -54,6 +55,9 @@ func ExecuteStateTransitionNoVerifyAnySig(
 	defer span.End()
 	var err error
 
+	interop.WriteBlockToDisk(signed, false)
```

Comment:
What is this false?

Reply:
Because the block has not failed

```go
func WriteBlockToDisk(block *ethpb.SignedBeaconBlock, failed bool) {
	filename := fmt.Sprintf("beacon_block_%d.ssz", block.Block.Slot)
	if failed {
		filename = "failed_" + filename
	}
}
```